From edfee40078d35223681f2c5e61c0582d38ea61be Mon Sep 17 00:00:00 2001
From: Michael Czigler <michaelczigler@icloud.com>
Date: Tue, 26 May 2020 09:18:10 +0000
Subject: [PATCH] Added scroll support.

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 293e00c..f369776 100644
--- a/config.def.h
+++ b/config.def.h
@@ -19,7 +19,7 @@ static int borderpx = 2;
 static char *shell = "/bin/sh";
 char *utmp = NULL;
 /* scroll program: to enable use a string like "scroll" */
-char *scroll = NULL;
+char *scroll = "scroll";
 char *stty_args = "stty raw pass8 nl -echo -iexten -cstopb 38400";
 
 /* identification sequence returned in DA and DECID */
-- 
2.26.2
